packages/core/src/avatar/Avatar.tsx

```diff
@@ -72,7 +72,11 @@ export const Avatar = forwardRef<HTMLDivElement, AvatarProps>(function Avatar(
     children = childrenProp;
   }
 
-  const avatarInitials = nameToInitials(name);
+  const avatarInitials = nameToInitials(name) && (
```

Comment:
I feel like this short-circuiting makes the code hard to read. If this was split out it would be easier to follow the logic.